ddp-workspace/projects/ddp-sdk/src/lib/components/cookies/cookiesPreferencesModal/cookiesPreferencesModal.component.ts

```diff
@@ -0,0 +1,106 @@
+import { Component, Inject } from '@angular/core';
+import { MatDialogRef, MAT_DIALOG_DATA, MatDialog } from '@angular/material/dialog';
+import { Cookies } from '../../../models/cookies';
+import { PrivacyPolicyModalComponent } from '../privacyPolicy/privacyPolicyModal.component';
+import { NoopScrollStrategy } from '@angular/cdk/overlay';
+
+@Component({
+  selector: 'ddp-cookies-preferences-modal',
+  template: `
+    <div class="cookiesModal cookiesModal__header">
+      <img class="cookiesModal__logo" lazy-resource src="/assets/images/project-logo-dark.svg"
+           [attr.alt]="'Toolkit.Common.LogoAlt' | translate">
```

Comment:
The Toolkit.* strings should only be used in toolkit and sometimes toolkit-prion.  If you need to use something in the SDK, put it in the SDK section.

Reply:
Done in baa19cc

Fixed for both modals